src/project/project-actions.tsx

```diff
@@ -1049,6 +1062,11 @@ export class ProjectActions {
       { changeName: change.name }
     );
   };
+
+  private isChromeOS105(): boolean {
+    const userAgent = navigator.userAgent;
+    return /(CrOS)/.test(userAgent) && /(Chrome\/105)/.test(userAgent);
```

Comment:
```suggestion
    return /CrOS/.test(userAgent) && /Chrome\/105\b/.test(userAgent);
```

I think the round brackets serve no purpose. Add \b (word boundary) so, um, I don't need to worry about Chrome 1050 (!)